pkg/api/v1beta1/types.go

```diff
@@ -412,6 +414,33 @@ type ContainerStatus struct {
 	ContainerID string `json:"containerID,omitempty" description:"container's ID in the format 'docker://<container_id>'"`
 }
 
+type PodConditionKind string
+
+// These are valid conditions of pod.
+const (
+	// PodReady means the pod is able to service requests and should be added to the
+	// load balancing pools of all matching services.
+	PodReady PodConditionKind = "Ready"
+)
+
+type PodConditionStatus string
```

Comment:
Okay will do. I was wondering if that qualified as an API break this weekend. I'll make the changes when I rebase over #4241.